imageprocessor/imageprocessor-core/src/main/java/pl/edu/uj/kimage/Image.java

```diff
@@ -0,0 +1,47 @@
+package pl.edu.uj.kimage;
+
+public final class Image {
+    private final Color[] data;
+    private final int width;
+    private final int height;
+
+    public Image() {
```

Comment:
I have added in only to compatibility with other code (there was already use of Image with empty constructor so far).